src/Midi_hook.h

```diff
@@ -39,8 +38,15 @@ class MidiHook : public QObject {
 	/// <summary>
 	/// Function pointer to execute action
 	/// </summary>
-	typedef void (*obsc)(MidiHook*);
+	typedef void (*obsc)(MidiHook *);
 	obsc obsControlFunction;
 	void EXE();
 	void setAction();
+	void initHotkey();
+	void setHotkey(obs_hotkey_t *hotkey);
+	obs_hotkey_t *getHotkey() const;
+
+private:
+	QString hotkey;
```

Comment:
Why is this QString private?  also if we are getting the hotkey strings from the hotkey object, why does this exist?

Reply:
The thing is, when you load config on startup, we have only hotkeyName stored. OBS Hotkeys are not initialized, when obs-midi config is loaded. Therefore we are not able to se obs_hotkey_t reference to MidiHook. This is done later, in `FinishedLoading` Event. But to be able to initialize a hotkey, we need the name of hotkey loaded from config. So this is the place, where I store the hotkeyName for this purpose.
It is private, so we don't change it by accident

Reply:
Got to make it private, so I can display which hotkey was not available on execution